src/test/resources/data/typicalDataBundle.json

```diff
@@ -149,6 +149,7 @@
       "courseId": "idOfTypicalCourse1",
       "name": "Instructor1 Course1",
       "email": "instructor1@course1.tmt",
+      "key": "instructor1@course1.tmt%idOfTypicalCourse1739021015",
```

Comment:
This works, but is not encouraged. To bypass the mismatch key, the correct thing to do is to ignore the key altogether.